curious/graph.py

```diff
@@ -153,7 +153,7 @@ def get_related_objects(instances, filters=None):
       queryset = queryset.extra(select={INPUT_ATTR_PREFIX: '%s' % rel_column})
 
     # M2M from instance to related objects
-    elif type(rel_obj_descriptor) in (ReverseManyRelatedObjectsDescriptor, ManyRelatedObjectsDescriptor):
+    elif type(rel_obj_descriptor) in (ReverseManyToOneDescriptor, ManyToManyDescriptor):
```

Comment:
isinstance?

Reply:
We actually don't want isinstance because that checks for subclasses so I'm going to play it safe and keep type